.travis.yml

```diff
@@ -1,14 +0,0 @@
-language: php
-
-sudo: false
-
-php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
```

Comment:
Why removing the Travis config file entirely ?

Reply:
Do you still use Travis? To me it's dead. Of course, this is your choice.